features/Hotels/Tc-20.feature

```diff
@@ -0,0 +1,11 @@
+#TC20
+Feature: TermsAndConditions
```

Comment:
You can have 1 feature file for all sprint 1 test cases.
Feature file should look like plain English not like variable name.